chore(web): use axios isCancel function

uhthomas opened the pull request:
Use of this function should be more idiomatic and reliable - it's much less likely for changes to Axios to cause regressions.

jrasm91 reviewed (approved):
Is this the only place it is used/checked?

uhthomas commented:
> Is this the only place it is used/checked?

I believe so - I don't see any other references.

https://github.com/search?q=repo%3Aimmich-app%2Fimmich%20CanceledError&type=code

![image](https://github.com/immich-app/immich/assets/9749173/39b75342-bc7b-4a00-9740-b03c65b4089e)

uhthomas committed:
chore(server): use axios isCancel function

Use of this function should be more idiomatic and reliable - it's much less
likely for changes to Axios to cause regressions.
